src/main/java/uk/gov/companieshouse/overseasentitiesapi/interceptor/UserAuthenticationInterceptor.java

```diff
@@ -69,20 +69,38 @@ public boolean preHandle(@NonNull HttpServletRequest request, @NonNull HttpServl
         final var tokenPermissions = getTokenPermissions(request)
                 .orElseThrow(() -> new IllegalStateException("UserAuthenticationInterceptor - TokenPermissions object not present in request"));
         String companyNumberInScope = getCompanyNumberInScope(request);
-        boolean isRoeUpdateJourneyRequest = isRequestRoeUpdate(companyNumberInScope);
```

Comment:
I think we should keep the `isRequestRoeUpdate` method as it makes the logic easier to read. It will now become more complex, however, as it will need to check the company number in the transaction.

Reply:
Since discussed, maybe, rather than try and detect if on the update journey, check if on the registration journey. Makes checks simpler, hopefully.